readme.md

```diff
@@ -49,3 +65,10 @@ See [process](./substrate-node/upgrade_process.md)
 ### Client
 
 You can use the client to interact with the chain, [read more](./cli-tool/readme.md)
+
+### Data Cleanup:
+To wipe data run:
+
+```
+rm -rf /home/<username>/.local/share/tfchain/chains/
```

Comment:
Paths are not the same on different systems, maybe we can add the official way to purge chain data:
```
./target/release/tfchain purge-chain --dev
```